### PR TITLE
Optionally unescape new line characters for nice display in LinePanel

### PR DESCRIPTION
**Pull request by Rictus**
See https://github.com/Textualize/toolong/issues/49

### CODE REVIEW

File: src/toolong/line_panel.py [hunk 1]
```diff
@@ -42,6 +43,8 @@ def compose(self) -> ComposeResult:
         else:
             yield Static(JSON.from_data(json_data), expand=True, classes="json")
             return
+        if self.app.new_lines:
+            self.text._text[0] = re.sub("\\\\n", "\n", self.text._text[0])
```
Comment (Rictus) [inline src/toolong/line_panel.py:47]: @willmcgugan  I'm not particularly happy with this, there may be a better way to do this using Textual primitives

**Comment by willmcgugan** [comment]
Any change you could give me a log file with escaped new lines?

**Comment by Rictus** [comment]
Sure,
[logs.log](https://github.com/Textualize/toolong/files/15132183/logs.log)

**Comment by willmcgugan** [comment]
Thanks. Added this to 1.5.0

**Pull request closed without merging**